python_ta/reporters/plain_reporter.py

```diff
@@ -5,7 +5,7 @@
 from collections import defaultdict, namedtuple
 from .node_printers import LineType, render_message
 
-OUTPUT_FILENAME = 'pyta_output'
+OUTPUT_FILENAME = 'pyta_output.html'
```

Comment:
@simonchenwastaken so it's not great to change this extension because for the `Plain` and `Color` reporter, the output is plaintext, not HTML.

What I recommend doing is moving this constant into the `PlainReporter` class itself, so that in the methods you can access `self.OUTPUT_FILENAME` instead. Then, this constant can be overridden in `HTMLReporter`.